### PR TITLE
Pull cloud guide to tensorflow.org and make some changes to the noteb…

### PR DESCRIPTION
**Pull request by yashk2810**
…ook.

### CODE REVIEW

File: guides/training_keras_models_on_cloud.py [hunk 1]
```diff
@@ -202,9 +202,9 @@ def create_model():
     callbacks = callbacks
     batch_size = 128
 else:
-    epochs = 1
-    callbacks = None
-    batch_size = None
+    epochs = 10
+    batch_size = 64
+    callback = None
```
Comment (fchollet) [inline guides/training_keras_models_on_cloud.py:207]: This should definitely be `callbacks`

Also, 1 epoch should be enough to debug

Comment (yashk2810) [reply guides/training_keras_models_on_cloud.py:207]: Done.

Comment (yashk2810) [reply guides/training_keras_models_on_cloud.py]: Keeping epochs at 5 so that the output on tf.org doesn't look sad

**Comment by yashk2810** [comment]
> Can you regenerate the .md file and other .ipynb file too?

I am getting some errors while generating the guide. I'll leave that to Francois to update.

**Pull request merged**